pallets/pallet-bonded-coins/src/types.rs

```diff
@@ -45,47 +46,46 @@ pub struct PoolDetails<AccountId, ParametrizedCurve, Currencies> {
 
 impl<AccountId, ParametrizedCurve, Currencies> PoolDetails<AccountId, ParametrizedCurve, Currencies>
 where
-	AccountId: PartialEq,
+	AccountId: PartialEq + Clone,
 {
-	pub fn new(
-		manager: AccountId,
-		curve: ParametrizedCurve,
-		bonded_currencies: Currencies,
-		transferable: bool,
-		state: PoolStatus<Locks>,
-	) -> Self {
+	pub fn new(owner: AccountId, curve: ParametrizedCurve, bonded_currencies: Currencies, transferable: bool) -> Self {
 		Self {
-			manager,
+			manager: Some(owner.clone()),
+			owner,
 			curve,
 			bonded_currencies,
 			transferable,
-			state,
+			state: PoolStatus::Locked(Default::default()),
```

Comment:
This should be the `Default` implementation of `PoolStatus`, I'd say.

Reply:
Yeah guess I'll just cherry-pick 9ae15d2310bb6a7f53b8e57cf7e972820f5f674b then. I had it implemented that way, can't tell you when exactly this disappeared.

Reply:
835cfeacce6711ee77758a5db6e75b59fa690cf2